collection-scripts/gather_ceph_resources

```diff
@@ -384,6 +383,28 @@ for ns in $namespaces; do
                     } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status_"${image}".part &
                     pids_rbd+=($!)
                 done
+                # Collecting mirror image status for provider mode with rados namespace
+                rados_namespaces=$(timeout 60 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd -p $bp namespace ls") 
+                for rns in $rados_namespaces; do
+                    {
+                        printf "Collecting mirror pool status for: %s\n" "${bp}"
+                        timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool status --verbose $bp --namespace ${rns} --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-status-"${bp}"-debug.log
+                    } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_status &
```

Comment:
The output for each namespace seems to be appended to the same file which is also used for the case of no namespace (default namespace).  Since per-namespace commands are run concurrently, the file would likely get corrupted.

I'd request that this PR is exercised on a setup where mirroring is configured on the default namespace and at least 10 custom namespaces before merging.